pegasus/src/social_metadata.rb

```diff
@@ -91,22 +86,19 @@ def get_social_metadata_for_page(request)
     },
     "hourofcode.com" => {
       "soon-hoc" => {
-        title: hoc_s(:social_hoc2018_hoc_coming_create),
-        description: hoc_s(:social_hoc2018_creativity_in_are_you),
-        image: images[:hoc_2018_creativity],
-        video: videos[:hour_of_code_worldwide]
+        title: hoc_s(:social_hoc2018_hoc_here),
```

Comment:
2019 here?

Reply:
Oh, this was deliberate because we happen to have the desired translated string ("The Hour of Code is here!") already.